test/integration/voldemort/performance/benchmark/Benchmark.java

```diff
@@ -642,6 +651,18 @@ public static void main(String args[]) throws IOException {
               .withRequiredArg()
               .describedAs(LOCAL_SERVER_PROPERTIES)
               .ofType(String.class);
+        parser.accepts(WORKLOAD_CLASS, "workload class (full name)")
+              .withRequiredArg()
+              .describedAs(WORKLOAD_CLASS)
+              .ofType(String.class);
```

Comment:
ok, does the following look good to you? I did use reflection to load the workload class anyway. 

parser.accepts(WORKLOAD_TYPE,
                       "class of the workload to use [optional, default is voldemort.performance.benchmark.DefaultWorkload]")